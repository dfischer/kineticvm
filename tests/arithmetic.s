    5 -> %LHS
    -3 -> %RHS
    %SUM -> %OUT ; expect 2
    %DIFF -> %OUT ; expect 8
    %PRODUCT -> %OUT ; expect -15
    %QUOTIENT -> %OUT ; expect -2 (rounded)
    3 -> %RHS
    %QUOTIENT -> %OUT ; expect 2 (rounded)
    2 -> %RHS
    %QUOTIENT -> %OUT ; expect 3 (round away from 0)
    -2 -> %RHS
    %QUOTIENT -> %OUT ; expect -3 (round away from 0)
    