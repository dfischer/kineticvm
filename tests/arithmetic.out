2
8
-15
-2
2
3
-3
5
-3
